src/popup/components/NetworkButton.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <BtnPill
+    :to="{ name: 'network-settings' }"
+    class="network-button"
+    narrow
+    dense
+  >
+    <div
+      class="circle"
+      :class="[ nodeStatus ]"
+    />
+    {{ activeNetwork.name }}
+  </BtnPill>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+import { useGetter, useState } from '../../composables/vuex';
+import { INetwork } from '../../types';
+
+import BtnPill from './buttons/BtnPill.vue';
+
+export default defineComponent({
+  components: {
+    BtnPill,
+  },
+  setup() {
+    const nodeStatus = useState('nodeStatus');
+    const activeNetwork = useGetter<INetwork>('activeNetwork');
+
+    return {
+      activeNetwork,
+      nodeStatus,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../styles/variables';
+@use '../../styles/typography';
+
+.network-button {
+  @extend %face-sans-14-medium;
+
+  margin-right: 6px;
+  margin-left: auto;
+  color: rgba(variables.$color-white, 0.75);
+  background: none;
+
+  .circle {
+    width: 6px;
+    height: 6px;
+    border-radius: 100%;
+    margin-right: 4px;
+    background-color: variables.$color-warning;
+
+    &.connected {
+      background-color: variables.$color-success-dark;
+    }
+
+    &.error {
+      background-color: variables.$color-danger;
+    }
+  }
+
+  &:hover {
+    color: rgba(variables.$color-white, 1);
+    background: variables.$color-bg-6;
+  }
+
+  &:active {
+    color: rgba(variables.$color-white, 0.75);
+    background: variables.$color-disabled;
+  }
```

Comment:
I think this should be removed as the hover and active states are managed by the BtnPill.